webapp/packages/plugin-sql-editor-navigation-tab-script/src/bindings/KEY_BINDING_SCRIPT_SAVE.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createKeyBinding } from '@cloudbeaver/core-view';
+
+export const KEY_BINDING_SCRIPT_SAVE = createKeyBinding({
+  id: 'script-save',
+  keys: ['ctrl+s'],
+  preventDefault: true,
+});
```

Comment:
you can add this binding to `webapp\packages\core-view\src\Action\KeyBinding\Bindings` (and remove `script` from id)

Reply:
also, can you add mac specific keybinding, please?